tests/components/yeelight/test_config_flow.py

```diff
@@ -67,9 +67,8 @@ async def test_discovery(hass: HomeAssistant):
         result3 = await hass.config_entries.flow.async_configure(
             result["flow_id"], {CONF_DEVICE: ID}
         )
-
     assert result3["type"] == "create_entry"
-    assert result3["title"] == NAME
+    assert result3["title"] == UNIQUE_NAME
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)